js/hangman.js

```diff
@@ -178,40 +180,36 @@ var Hangman = (function ($, window) {
       word = word.replace("ß", "ss");
       return { word: word, level: wordLevel(word) };
     });
+    //console.log(allWords);
     $("#n-words").text(allWords.length);
     newGame();
   }
 
   function wordLevel(word) {
-    var i, pos;
+    var i, x;
+    var level = 0;
     var wordValue = 0;
     var chars = [];
-    word = word.toLowerCase();
+    //console.log(WordValueThresholds.length);
     word = word.split("");
     for (i = 0; i < word.length; ++i){
-      wordValue = wordValue + LetterValue[word[i]];
-      pos = 0;
+
       if (jQuery.inArray(word[i],chars) === -1) {
         chars.push(word[i]);
+        wordValue = wordValue + LetterValue[word[i]];
       }
     }
-    if (chars.length < 4) {
-      wordValue = wordValue + 5;
-    }
-    else if (chars.length < 7) {
-      wordValue = wordValue + 50;
-    }
-    else if (chars.length < 10) {
-      wordValue = wordValue + 100;
-    }
-    else {
-      wordValue = wordValue + 150;
+    x = (word.length + chars.length*2) / 3;
+    wordValue = wordValue * ((2.84138*Math.pow(10,-4)*Math.pow(x,4))+(-0.0174815347*Math.pow(x,3))+(0.3999311992*Math.pow(x,2))+(-4.118219251*x)+(16.93973598));
+    for (i = 0; i < WordValueThresholds.length; i++) {
```

Comment:
As you can see in other parts of my code, unary operators like `--` or `++` should precede the identifier. Thanks for your attention.